msg/BoundedSequencesNoStrings.msg

```diff
@@ -0,0 +1,32 @@
+# Bounded sequences of different types
```

Comment:
@MiguelCompany if the core idea behind adding messages is to bounded sequence of plain data types, consider making that explicit in the name e.g. `BoundedPlainSequences`, `BoundedPODSequences` or any meaningful name you like.